packages/server/routes/api/events.js

```diff
@@ -114,4 +114,48 @@ router.get(
   }
 );
 
+router.get(
+  "/eventAttendees/:eventId",
+  AuthController.authenticate,
+  async (request, response) => {
+    const eventId = request.params.eventId;
+
+    if (!validate(eventId)) {
+      response.status(400).json({
+        status: "fail",
+        data: {
+          eventId: `Given eventId "${eventId}" is invalid.`,
+        },
+      });
+      return;
+    }
+    if (!(await EventController.existsInPrisma(eventId))) {
+      response.status(404).json({
+        status: "fail",
+        data: {
+          eventId: `Given eventId "${eventId}" does not exist.`,
+        },
```

Comment:
The error message here would be clearer with something like `No event exists with an ID of ${eventId}`.